src/carriers/mjpeg_carrier/MjpegDecompression.cpp

```diff
@@ -12,6 +12,7 @@
 
 #include <csetjmp>
 #include <cstdio>
+#include <string.h>
```

Comment:
please use `cstring` instead of `string.h`

Reply:
Done https://github.com/robotology/yarp/pull/1462/commits/67ad2602ea649690eaac7a989a0cf504fdbb11e2